sklearn/inspection/_plot/decision_boundary.py

```diff
@@ -248,6 +250,14 @@ def from_estimator(
             For multiclass problems, :term:`predict` is selected when
             `response_method="auto"`.
 
+        class_of_interest : int, float, bool or str, default=None
```

Comment:
I'm not sure what `float` and `bool` here mean? do we accept floats for classification targets? is `bool` accepted when target is boolean?

Reply:
> do we accept floats for classification targets?

(unfortunatelly) yes we do. The type reported here are the same than the one accepted as `pos_label` in thresholded metric.